### PR TITLE
osu! Beatmapping World Championship 2020

### PR DESCRIPTION
**Pull request by Pachiru**
Newspost for the osu! Beatmapping World Championship 2020.

### CODE REVIEW

File: news/2020-05-02-beatmapping-world-championship-2020.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship #2"
+date: 2020-05-02 12:00:00 +0000
+---
+
+Welcome to the second edition of the osu! Beatmapping World Championship! Check out the changes to the contest format as well as everything else to look forward to!
+
+![](https://i.imgur.com/oqHbVX8.jpg)
```
Comment (venix12) [inline news/2020-05-02-beatmapping-world-championship-2020.md:9]: the banner should be put in ``wiki/shared/news/banners`` as an image instead i guess

Comment (Pachiru) [reply news/2020-05-02-beatmapping-world-championship-2020.md]: Yup, fixed it

File: news/2020-05-02-beatmapping-world-championship-2020.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+layout: post
+title: "osu! Beatmapping World Championship 2020"
+date: 2020-05-02 12:00:00 +0000
+---
+
+Welcome to the second edition of the osu! Beatmapping World Championship! Check out the changes to the contest format as well as everything else to look forward to!
+
+![](wiki\shared\news\banners\bwc2020.jpg)
+
+We're excited to announce the second osu! Beatmapping World Championship. As with the first edition, participants will be forming teams based on their country, but the contest format has some new additions to make things more exciting for both participants and audiences alike! Please read below for a summary of the contest format this time around:
```
Comment (Ephemeralis) [inline news/2020-05-02-beatmapping-world-championship-2020.md:11]:
```suggestion
As with the first edition, participants will be forming teams based on their country, but the contest format has some new additions to make things more exciting for both participants and audiences alike! 

Please read below for a summary of the contest format this time around:
```

File: news/2020-05-02-beatmapping-world-championship-2020.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Team Format
+
+Teams will be formed based on each country as designated by osu!. Teams may consist of 3-7 members, with individual participants registering to be eligible to be selected based on their country displayed on their osu! profile. Participants may also apply to be chosen as the *Team Captain*, who will be responsible for selecting the other team members as well as being the primary contact with the contest staff. Participant applicants will vote for who they would like to be the Team Captain for their team, with the winner being selected as the Captain. Please note that Captain applicants will be vetted by the contest staff prior to voting, with the expectation that they have mapping/modding experience in some form.
```
Comment (Ephemeralis) [inline news/2020-05-02-beatmapping-world-championship-2020.md:15]:
```suggestion
Teams will be formed based on each country as designated by osu!. Teams may consist of 3-7 members, with individual participants registering to be eligible to be selected based on their country displayed on their osu! profile. 

Participants may also apply to be chosen as the *Team Captain*, who will be responsible for selecting the other team members as well as being the primary contact with the contest staff. Participant applicants will vote for who they would like to be the Team Captain for their team, with the winner being selected as the Captain. 

Please note that Captain applicants will be vetted by the contest staff prior to voting, with the expectation that they have mapping/modding experience in some form.
```

**Comment by Pachiru** [comment]
Applied changes!

**Comment by nl-tatatat** [comment]
Can you please make it painfully obvious what gamemode this is for in the news post? I see no text describing it, but I assume its only for osu.

File: news/2020-05-02-beatmapping-world-championship-2020.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship 2020"
+date: 2020-05-02 12:00:00 +0000
+---
+
+Welcome to the second edition of the osu! Beatmapping World Championship! Check out the changes to the contest format as well as everything else to look forward to!
+
+![](wiki\shared\news\banners\bwc2020.jpg)
```
[the remark this answers is not in the document]
Comment (Pachiru) [reply news/2020-05-02-beatmapping-world-championship-2020.md]: Tried to apply but doesn't look like it's changing anything

**Pull request merged**